### ISSUE TITLE
Cannot stop a running scan

### ISSUE THREAD

**Issue by bob3rocks** [smicallef/spiderfoot#1745]
Today I started using SpiderFoot v4.0.0, which came bundled with telekom-security/tpotce honeypot installation.

What version of Python are you using? Python 3.8.10

I am running an `All` use case scan on a public IP address which I own.

`Debug` is turned off; nonetheless, logs sent to /var/log/syslog are verbose and causing file bloat.

Trying to stop a running scan, I get the green `scan aborted` popup -- but the scan never stops.

Trying to delete the scan, cannot delete a running scan.

I know there is another issue already open for this, but that is a year old and could be a separate issue.

**Comment by winschdi** [comment]
Yep, exact same issue here, I'm also using t-pot with spiderfoot v4.0.0

**Comment by TurhanErgene** [comment]
If you are using the web UI, navigate to Scans and click the stop button under the Action column. If somehow still not working, try:
ps aux | grep spiderfoot
kill -9 <PID>
